src/optimization/utils/constraints.jl

```diff
@@ -103,8 +104,15 @@ function encode_layer!(SLP::SlackLP, model::Model, layer::Layer{Id}, ẑᵢ, z
     return nothing
 end
 
+# need to fix δᵢⱼ for BoundedMixedIntegerLP and possibly other types 
```

Comment:
would rather delete this comment

Reply:
don't see why I think it adds some much needed documentation

Reply:
It doesn't need to be fixed anymore, because you fixed it 😜